Testreglar/252/Nett/nett-2.5.2a.json

```diff
@@ -0,0 +1,189 @@
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Slipphandlingen (up-event) opphever ikke resultatet av den foregående trykkhandlingen (down-event)"
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Fullfører slipphandlingen en funksjon?",
+			"ht": "<p>Eksempel:</p>\n<ul>\n<li>Dra- og slipp: Brukeren flyttet et objekt til sluttpunktet</li>\n<li>Slette fil eller data: Brukeren slettet en fil eller data</li>\n<li>Aktivere en funksjonalitet: Brukeren aktiverte høykontrastvisning av en nettside</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Slipphandlingen (up-event) opphever resultatet av den foregående trykkhandlingen (down-event)"
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				}
+			}
+		},
+		{
+			"stegnr": "3.6",
+			"spm": "Finnes det en mekanisme for å angre funksjonen etter at den er fullført, når funksjonen fullføres på slipphandlingen?",
+			"ht": "<p>Eksempel: </p>\n<ul>\n<li>Dialogboks for å bekrefte, avbryte eller brukeren kan flytte et objekt tilbake til utgangspunktet</li>\n<li>Dialogboks for å bekrefte, avbryte eller brukeren kan angre sletting av en fil</li>\n<li>Angre-knapp eller lignende</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Det finnes ikke en mekanisme for å angre funksjonen fullført av slipphandlingen (up-event)."
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Fungerer mekanismen som skal angre funksjonen? ",
+			"ht": "<p>Eksempel:</p>\n<ul>\n<li>Dra- og slipp: brukeren flyttet et objekt fra et startpunkt til sluttpunkt, det finnes en mekanisme for å flytte objektet tilbake til startpunkt.</li>\n<li>Sletting: brukeren slettet et dokument eller en mappe, det finnes en mekanisme for å gjenopprette filen.</li>\n<li>Endret visning: brukeren aktiverte høykontrast versjon av en nettside, det finnes en mekanisme for å endre til vanlig versjon. </li>\n<li>Andre lignende mekanismer som angrer funksjonen som ble utført på slipphandlingen etter at den er fullført</li>\n</ul>",
```

Comment:
"Endret visning: brukeren aktiverte høykontrast versjon av en nettside, det finnes en mekanisme for å endre til vanlig versjon" 

Er "refresh" en godkendt mekanisme til dette? 

Eller skal det være en mekanisme som stilles til rådighed via en brugergrænsefladekomponent? 

Og skal denne mekanisme være tilgængelig for at være godkendt?

Reply:
Basert på definisjonen til mekanisme, den kan få støtte fra brukeragenter og refresh er et eksempel på dette. Vi pleier ikke å være veldig spesifikk og quick/fix når vi gir eksempler i våre testregler.